kopi/chain.json

```diff
@@ -0,0 +1,270 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Kopi",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://app.kopi.money",
+  "pretty_name": "Kopi",
+  "chain_type": "cosmos",
+  "chain_id": "luwak-1",
+  "daemon_name": "kopid",
+  "node_home": "$HOME/.kopid",
+  "bech32_prefix": "kopi",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md